src/view/searchview.cpp

```diff
@@ -95,6 +95,11 @@ auto SearchView::defaultTree(const QStringList &headers) -> QTreeWidget *
 	return tree;
 }
 
+void SearchView::focusSearchBox(void)
+{
+	searchBox->setFocus();
```

Comment:
Doesn't it make more sense to just put it in a `showEvent`? It would also avoid needing the forward declare.

Reply:
I agree.